package.json

```diff
@@ -10,7 +10,7 @@
     "url": "git://github.com/strongloop/strong-docs.git"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "mocha --timeout 15000"
```

Comment:
This is causing failure on Jenkins (Mac). I've run into the issue before on other repos (each run was different and I ended up having to disable timeout completely `this.timeout(0)`. The machine is really slow but maybe we can bump this up for CI ... or just trigger it again till it passes.